### DISCUSSION TITLE
controller 状态管理

### DISCUSSION THREAD

**Discussion by amberlwan** [react-hook-form/react-hook-form#7177] [Q&A]
两个controller 包含的组件，用了表单中的同一个字段，更新其中一个的值，另外一个render中拿到的值还是旧的，我看了下controller也是用usewatch实现的，为什么会出现这个情况呢

**Comment by bluebill1049** [comment]
can you provide a codesandbox?

**Comment by amberlwan** [comment]
比如可以看下这个简单事例https://codesandbox.io/s/react-hook-form-error-v6-forked-1p8vc?file=/src/index.js，两个controller的状态不同步




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "react-hook-form/react-hook-form"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年11月30日(星期二) 中午1:30
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [react-hook-form/react-hook-form] controller 状态管理 (Discussion #7177)





 
can you provide a codesandbox?
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
